dashboard/test/ui/features/star_labs/applab/level_options.feature

```diff
@@ -1,6 +1,6 @@
-@as_student
```

Comment:
note about this change: only 1 of the 2 scenarios in this file was actually using this decorator (the first one), so i moved it to the relevant scenario to remove the unused extra step (especially because this decorator makes a db call to create a student)